config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  get "/items/my", to: "items#my_items"
+  get "/items/borrowed", to: "items#borrowed_items"
```

Comment:
It feels a bit weird to extend `/items` like this, given it's declared as a resource. @luap42 what would be the Rails-idiomatic way of doing this?